pages/changelog.mdx

```diff
@@ -21,11 +21,17 @@ Keep up to date on new product features and enhancements, API docs, tools change
 
 ## Completed
 
+### 2022-10-31
+
+**ADDED**
+
+- Added support for including [facilitator fee](/concepts/facilitator-fee) when creating transfers from Verified Customer's Bank into their Dwolla balance.
```

Comment:
smol change to pluralize. Change facilitator fee to `facilitator fees`

Reply:
Updated [here](https://github.com/Dwolla/dev-portal/pull/439/commits/08d8a1f9dfa6c889f3e976111cac931891b39478).